week1/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+  "bufio"
+  "fmt"
+  "os"
+  "strconv"
+  "strings"
+)
+
+func eval(text string) {
```

Comment:
In my opinion, we should separate the code doing calculation with the code printing output to stdout. It's easier to unit-test this way.